subsys/usb/class/Kconfig

```diff
@@ -101,24 +101,25 @@ config USB_DEVICE_BLUETOOTH
 	help
 	  USB Bluetooth device class driver
 
+if USB_DEVICE_BLUETOOTH
+
 config USB_DEVICE_BLUETOOTH_VS_H4
 	bool "Enable USB Bluetooth H4 vendor command"
-	depends on USB_DEVICE_BLUETOOTH
```

Comment:
Please do not remove `depends on USB_DEVICE_BLUETOOTH`, it makes no sense to remove two lines and then insert four to archive the same thing.